admin/assembly-rhdh-telemetry.adoc

```diff
@@ -0,0 +1,42 @@
+[id='assembly-rhdh-telemetry_{context}']
+
+= Telemetry data collection
+
+You can use the telemetry data collection feature by configuring the `analytics-provider-segment` plugin, which helps in sending analytics events to a specific Segment. For example, if there are number of users accessing your application, you can create multiple Segments based on location, purchasing tendency, or visiting pattern of your application users. So, a Segment denotes a classified set of your analytics data.
+
+
+Based on the data collected, {company-name} analyzes your usage of the {product} application. This analysis helps in providing you support for upgrades, optimizing performance or configuration, minimizing service impacts, identifying and remediating threats, troubleshooting, improving the offerings and your experience, and responding to your issues. 
+
+This feature collects the following data about your {product-very-short} usage:
+
+* Events-related information, for example,  number of page visits, number of clicks on any links or buttons.
+* System-related information, for example, locale, timezone, user agent including browser and OS details.
+* Page-related information, for example, title, category, extension name, URL, path, referrer, and search parameters.
+
+This feature also helps in anonymizing IP address and username-related information to maintain your privacy.
+
+* The IP addresses are anonymized and recorded as `0.0.0.0`.
+* The usernames are recorded as an anonymized hash ID.   
+
+The anonymized IP addresses and usernames denote a unique random identifier that is used solely to identify the number of unique users of the {product-very-short} application.
+
+{company-name} analyzes the collected data to understand your interactions with the {product-very-short} application without disclosing your identity and compromising your privacy. {company-name} uses the collected data only for internal analysis and to improve your experience with {product}.
+
+With {product-very-short}, you can customize the telemetry data collection feature and the telemetry Segment source configuration based on your needs.
+
+[IMPORTANT]
+====
+You can also use the telemetry data collection feature in an air-gapped environment in the same way as described in the following procedures.
```

Comment:
@kadel @kim-tsao, Should we close [this ticket](https://issues.redhat.com/browse/RHIDP-1909), wherein we had to capture the procedure to disable telemetry in an airgap scenario? Because it is impractical to use telemetry in air-gapped environments.